src/controllers/boolController.js

```diff
@@ -0,0 +1,26 @@
+const { negate, truthiness, isOdd, startsWith } = require('../lib/booleans');
+
+exports.bool_negate = (req, res) => {
+  res.status(200).json({ result: negate(req.body.value) });
+};
+
+exports.bool_truthiness = (req, res) => {
+  res.status(200).json({ result: truthiness(req.body.value) });
+};
+
+exports.bool_isOdd = (req, res) => {
+  const number = parseInt(req.params.number);
+
+  return Number.isNaN(number)
+    ? res.status(400).json({ error: 'Parameter must be a number.' })
```

Comment:
I like the use of ternaries here. Additional info comment - Depending on the company you work for in future and the style of code they want you to write in, consider whether it is clear what is happening if you had never seen this code before. Is it the most human readable way of reading the code, is it the most efficient way. These are considerations you will start to make as your career progresses based on the priorities your company gives you :)